changelog_entry.yaml

```diff
@@ -0,0 +1,10 @@
+- bump: major
+  changes:
+    added:
+    - /tracer_analysis endpoint for household tracer outputs
+    - Database for simulation tracer outputs
+    changed:
+    - /analysis endpoint renamed to /simulation_analysis endpoint
```

Comment:
Can we rename the endpoints to use [hyphens](https://stackoverflow.com/questions/10302179/hyphen-underscore-or-camelcase-as-word-delimiter-in-uris), not underscores?

<img width="729" alt="image" src="https://github.com/user-attachments/assets/d868f2b5-d32a-4c0d-8cac-7025b2fe4067">